### PR TITLE
NRPS substrate specificities

### PR DESCRIPTION
**Pull request by BTheDragonMaster**
This PR updates and adds NRPS substrates to 430 previously unannotated clusters and updates 769 substrates from existing entries. Evidence codes and publications were added too. Furthermore, clusters with conflicting substrate specificities between original annotations and new annotations were manually curated and changed accordingly.

check_valid.py was run on all .jsons (with the updated schema), no errors were returned. All substrates (322 distinct molecules) now have valid SMILES strings associated with them (validated with PIKAChU). The temporary structure code '[Po][Po]', which was used for missing substrate structures, is no longer found in any .json file.

The json schema was updated to include extra evidence codes, and to change Aspartate and Glutamate to Aspartic acid and Glutamic acid respectively.

### CODE REVIEW

File: data/BGC0000002.json [hunk 1]
```diff
@@ -18,6 +18,11 @@
                 "AAAAAAAAAAAAAAAAAAAAAAAA"
             ],
             "version": "2.0"
+        },
+        {
+            "comments": [],
+            "contributors": [],
+            "version": "3.0"
```
Comment (SJShaw) [inline data/BGC0000002.json:25]: All these empty changelogs should be removed.

**Review by SJShaw — CHANGES_REQUESTED** [review]
There's a few things here:
- the many empty changelogs
- the newly inconsistent lower-case vs title-case of substrate names has me a bit puzzled. Is there some meaning to that system?
- the revert would be best rebased out (this would also apply to the empty changelogs above), especially since a rebase is required anyway to solve the merge conflicts

File: data/BGC0000002.json (hunk 1 above)
Comment (BTheDragonMaster) [reply data/BGC0000002.json]: Currently, substrate names are only capitalised when they are proteinogenic (mostly due to the check for proteinogenic AA names in the current json schema). I'd suggest to make everything lower-case for consistency, as auto-capitalisation of other substrates is not trivial.

Will rebase and remove empty changelogs.

Comment (kblin) [reply data/BGC0000002.json]: I agree, we should change the schema to not capitalise the amino acids. That's just been transferred in from the old schema, I guess.

**Comment by kblin** [comment]
This is now handled by #240

**Pull request closed without merging**